first-pull(markup)

KluDan opened the pull request:
rest In progress

KluDan committed:
first-pull(markup)

rest In progress

KluDan committed:
structural changes in filters files

Сhanged markup structure, fixed filters.css, cleaned repeating styles, added variables, transitions. Changed filters.js - added more functional, but main API queries In process

biggujo reviewed (changes requested):
Edit comment: not related

biggujo commented:
![](https://github.trello.services/images/mini-trello-icon.png) [відображення рецепців з урахуванням фільтрів](https://trello.com/c/b6oK2dKf/41-%D0%B2%D1%96%D0%B4%D0%BE%D0%B1%D1%80%D0%B0%D0%B6%D0%B5%D0%BD%D0%BD%D1%8F-%D1%80%D0%B5%D1%86%D0%B5%D0%BF%D1%86%D1%96%D0%B2-%D0%B7-%D1%83%D1%80%D0%B0%D1%85%D1%83%D0%B2%D0%B0%D0%BD%D0%BD%D1%8F%D0%BC-%D1%84%D1%96%D0%BB%D1%8C%D1%82%D1%80%D1%96%D0%B2)

biggujo commented:
![image](https://github.com/biggujo/project-cooking/assets/37770603/84e200fc-5142-47bc-8c22-007297dbd022)

Fix simultaneous open of all <input>

KluDan committed:
Merge branch 'main' into origin/feature/filters

KluDan committed:
Fixed simultaneous open of all

fixed misspellings in html, css. Insert variables for animation.

KluDan committed:
Change requests in filters.js

added axios, then/catch;
alphabetical sorting of selects;
added reusable functions

KluDan committed:
Fixed resetFiltersEl

Fixed handler function to clear all filters active properties

KluDan committed:
Update filters.js

Added function to collect data from filters for query

KluDan committed:
added debounce for input

Fixed clear input function

KluDan committed:
Added filter requests

Added filter requests, so far for each separately.
